### PR TITLE
FAQ divider style fix

### PR DESCRIPTION
**Pull request by Adhithyarajan**
I've replaced <hr> with <div> and styled it using CSS.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -103,6 +103,7 @@ <h2>how it works!</h2>
           <div onclick="expand('Q1')" class="question">
             <button><img src="images/add.svg" id="Q1-icon"></button>
             Are the insights accurate?
+            <div class="divider"></div>
```
Comment (ajayesivan) [inline public/index.html:106]: This element has no effect in the design please remove it.

File: public/index.html [hunk 2]
```diff
@@ -111,40 +112,46 @@ <h2>how it works!</h2>
             insights. It's worth noting that the accuracy of the insights improves as you input more data into the
             system. Therefore, we recommend inputting your vehicle's expenses regularly and ensuring the information
             isaccurate and up-to-date to obtain the most precise insights possible.
-          </p><hr>
+          </p><div class="divider"></div>
```
Comment (ajayesivan) [inline public/index.html:115]: We only need one divider element after every `p` tag. This one looks good. Please move it to the next line for better readability.

File: public/styles.css [hunk 3]
```diff
@@ -174,6 +174,11 @@
   font-size: 60px;
 }
 
+.divider {
+  border-top: 1.5px solid #bc1717;
```
Comment (ajayesivan) [inline public/styles.css:178]: change border size from `1.5px` to `1px` and color to `#E6E6E6`

Please check the Figma **Inspect** tab to get accurate values.

File: public/styles.css [hunk 4]
```diff
@@ -174,6 +174,11 @@
...
+  margin-top: 10px 0;
```
Comment (ajayesivan) [inline public/styles.css:179]: margin-top only accept a single value.
```suggestion
  margin-top: 10px;
```

File: public/styles.css [hunk 5]
```diff
@@ -525,6 +530,11 @@ html {
     justify-content: center;
   }
 
+  .divider {
+    border-top: 1.5px solid #bc1717;
+    margin-top: 10px 0;
+  }
```
Comment (ajayesivan) [inline public/styles.css:536]: We only need to define the `divider` class once.

**Review by ajayesivan — CHANGES_REQUESTED** [review]
@Adhithyarajan Good start! I have added a few comments.

Please also do the following,
- Change the PR title to something meaningful, eg: Fixed terms and conditions page download section.
- Link the related issue to this PR.

File: public/styles.css [hunk 6]
```diff
@@ -174,6 +174,11 @@
   font-size: 60px;
 }
 
+.divider {
+  border-top: 1px solid #e6e6e6;
+  margin-top: 10px;
```
Comment (ajayesivan) [inline public/styles.css:179]: Add space before & after the divider, please check figma design for actual values.

File: .vscode/settings.json
Comment (ajayesivan) [inline .vscode/settings.json:1]: Remove this file from tracking. I think many of you have started using vs code live server. So, add `.vscode` to `.gitignore` file to make sure nothing from that folder gets tracked.

Reference: https://git-scm.com/docs/gitignore

**Review by ajayesivan — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (ajayesivan) [inline .vscode/settings.json:1]: @Adhithyarajan Please remove this file from tracking.

Comment (ajayesivan) [reply .vscode/settings.json:1]: @Adhithyarajan Did you update `.gitignore` file?

Comment (Adhithyarajan) [reply .vscode/settings.json:1]: OK

Comment (Adhithyarajan) [reply .vscode/settings.json:1]: Yes.

File: public/styles.css (hunk 6 above)
Comment (Adhithyarajan) [reply public/styles.css:178]: Can you please explain this issue or give a reference or something coz i've checked figma design but i couldn't get a desired output after replacing the values.

Comment (Adhithyarajan) [reply public/styles.css:178]: Can you please explain this issue or give me a reference or something coz i've replaced the values by checking figma but didn't get the desired output.

File: .vscode/settings.json
Comment (Adhithyarajan) [reply .vscode/settings.json:1]: Can you please check it.

**Review by ajayesivan — CHANGES_REQUESTED** [review]

**Comment by ajayesivan** [comment]
@Adhithyarajan Please also link the related issue.

**Review by ajayesivan — APPROVED** [review]
LGTM! Great Work @Adhithyarajan 🎉 

Thanks for the PR. You can go ahead and merge this using the GitHub **Squash & Merge** option.

**Pull request merged**